### ISSUE TITLE
error: unable to find 'data/"master".zig'

### ISSUE THREAD

**Issue by kxte** [zigtools/zls#144]
When attempting to build ZLS master, using Zig `0.6.0+cf4936bcb`, I encountered an error involving the data version flag. I assume it is caused by [this recent fix](https://github.com/ziglang/zig/commit/f23fb3087befc041161498a086bac2a92acea6cb) in Zig itself.

```sh
zig build -Drelease-safe -Ddata_version=master --prefix .
```
```zig
./src/main.zig:8:14: error: unable to find 'data/"master".zig'
const data = @import("data/" ++ build_options.data_version ++ ".zig");
             ^
./src/main.zig:965:29: note: referenced here
    const CompletionList = [data.builtins.len]types.CompletionItem;
                            ^
./src/main.zig:1204:34: note: referenced here
                        .items = builtin_completions[@boolToInt(use_snippets)][0..],
                                 ^
zls...The following command exited with error code 1:
/home/kate/.local/zig/zig build-exe /home/kate/.local/zls/src/main.zig --pkg-begin build_options /home/kate/.local/zls/zig-cache/zls_build_options.zig --pkg-end --release-safe --cache-dir /home/kate/.local/zls/zig-cache --name zls --pkg-begin known-folders /home/kate/.local/zls/src/known-folders/known-folders.zig --pkg-end --cache on

Build failed. The following command failed:
/home/kate/.local/zls/zig-cache/o/tln8XTMTk2Vqfxw-snqT2G51m9Zg3RXkbvBjnmMuB-Si3HzK6RxYW9iR_0VVKuig/build /home/kate/.local/zig/zig /home/kate/.local/zls /home/kate/.local/zls/zig-cache -Drelease-safe -Ddata_version=master
```

**Comment by SuperAuguste** [comment]
It is indeed caused by the Zig PR you mentioned! I’ll fix this as soon as I can.

**Comment by SuperAuguste** [comment]
This should now be fixed in 4f9f317374f6bc678723b49ed7a9e49612388dc8 @kxte!
